### PR TITLE
AUSTA-257: Cleanup and Copy Setup Samba Blueprint into Automate Windows Installation with autounattend

### PR DESCRIPTION
**Pull request by h4xhor**
AUSTA-257: Added the blueprint "Setup Samba on Linux Worker".

### CODE REVIEW

File: parameters/linuxrootuserdelete/metadata.json
Comment (brentonford) [inline parameters/linuxrootuserdelete/metadata.json:1]: @h4xhor, please remove this parameter

File: steps/installprerequisitesonlinuxworker/README.md
Comment (brentonford) [inline steps/installprerequisitesonlinuxworker/README.md:1]: @h4xhor, Are these required for the Samba Share or the Linux Worker?

File: steps/setupsambaonlinuxworker/README.md
Comment (brentonford) [inline steps/setupsambaonlinuxworker/README.md:1]: @h4xhor, the child steps under this Blueprint finish with "on Linux Worker". Can the Samba Share be it's own stand alone server?

**Comment by brentonford** [comment]
@h4xhor, please fix the Samba Share Blueprint and Step to reference a Linux Server and not the Linux Worker.

**Review by brentonford — COMMENTED** [review]
@h4xhor, please fix the Samba Share Blueprint and Step to reference a Linux Server and not the Linux Worker.

File: parameters/linuxrootuserdelete/metadata.json
Comment (h4xhor) [reply parameters/linuxrootuserdelete/metadata.json:1]: @brentonford I couldn't find this parameter but I found another `Linux: Attune User DELETE` to delete.
image

File: steps/installprerequisitesonlinuxworker/README.md
Comment (h4xhor) [reply steps/installprerequisitesonlinuxworker/README.md:1]: It's for the Samba share.

File: steps/setupsambaonlinuxworker/README.md
Comment (h4xhor) [reply steps/setupsambaonlinuxworker/README.md:1]: Yes.

**Pull request closed without merging**